### ISSUE TITLE
Compound report card fake sections

### ISSUE THREAD

**Issue by agaulton** [chembl/GLaDOS#508]
Probably you already know, but just to make sure captured... These sections currently still show fake data:
Indications
Clinical data
Target predictions
Compound cross references

**Comment by mnowotka** [comment]
Yes we do :) Antually I asked @nclopezo to produce a smilar list. Davis, can you please have a look and tell if there is anything more? Cross references will most probably be done before SAB, currently the data is missing in the API and I'm working on this. For the rest we need to discuss with David.

**Comment by nclopezo** [comment]
I recently added these sections:

- Biological Sequence (https://github.com/chembl/GLaDOS/pull/550)
- HELM Notation (https://github.com/chembl/GLaDOS/pull/548)
- Clinical Data (https://github.com/chembl/GLaDOS/pull/552)

The following sections are missing:

- Indications
- Target Predictions
- Cross References

I am currently working on those sections

**Comment by nclopezo** [comment]
Drug indications is now ready (#562):
https://chembl-glados.herokuapp.com/compound_report_card/CHEMBL25/#Indications

**Comment by nclopezo** [comment]
Target predictions is now ready (https://github.com/chembl/GLaDOS/pull/588): 
https://chembl-glados.herokuapp.com/compound_report_card/CHEMBL57/#TargetPredictions

**Comment by nclopezo** [comment]
Compound xefs is almost ready (#592).
If you compare, some sources are not there, they are not in the response from the web services:
- https://www.ebi.ac.uk/chembl/compound/inspect/CHEMBL25
- https://chembl-glados.herokuapp.com/compound_report_card/CHEMBL25/

Here, for example, ATC or ChemSpider don't come in the [web services response](https://www.ebi.ac.uk/chembl/api/data/molecule/CHEMBL25.json). Also, for some sources, there are no entries [here](https://www.ebi.ac.uk/chembl/api/data/xref_source.json). There is no entry for PubChem for example. so the link can not be built. Unless the base urls are hardcoded in the interface code, which I don't want to do. 
@mnowotka, @agaulton

**Comment by nclopezo** [comment]
Compound xrefs should be finished (https://github.com/chembl/GLaDOS/pull/593). What is missing now is the target xrefs section, and all the fake data should be gone.

**Comment by nclopezo** [comment]
The target xrefs section is not fake anymore:
https://chembl-glados.herokuapp.com/target_report_card/CHEMBL1991/
https://chembl-glados.herokuapp.com/target_report_card/CHEMBL3848/
However, it only shows what we have currently. After https://github.com/chembl/chembl_webservices_2/issues/154 is done I will finish this section.

**Comment by mnowotka** [comment]
chembl/chembl_webservices_2#154 is now done.

**Comment by nclopezo** [comment]
For the record, https://github.com/chembl/GLaDOS/issues/600 is almost done, this was one of the last 'fake data' sections. We still need to:

- [ ] Add some details to the similar documents section https://github.com/chembl/GLaDOS-es/issues/21.
- [ ] Make sure target components references are in the target components index. 
- [ ] Add target components xrefs to target report card. 

After this, there should be no more fake data in any report card, and this issue should be ready to be closed.

**Comment by mnowotka** [comment]
And:

- [ ] do connectivity matches for UniChem Cross References as described in #647

**Comment by nclopezo** [comment]
That is part of the tasks to do for https://github.com/chembl/GLaDOS/issues/647.
UniChem Cross Refs already has real data which I understand is the point of this issue.

**Comment by mnowotka** [comment]
It is but I think the spirit of the #647 is merging data of children into parents. All the cross-reference issues should be done within this task and then in #647 there will be only substask related to parent-child relation.

**Comment by nclopezo** [comment]
But the data is not fake, it's just missing, and #647 already describes that. It's like adding https://github.com/chembl/GLaDOS/issues/545 to the task list here when there is already an issue that describes that bug.

**Comment by agaulton** [comment]
Just in case it's not on the radar... There are still some fake icons (cats) in the molecule features section e.g., https://chembl-glados.herokuapp.com/compound_report_card/CHEMBL2108432/

**Comment by nclopezo** [comment]
The target xrefs section is now ready:

https://chembl-glados.herokuapp.com/target_report_card/CHEMBL1163125/

**Comment by agaulton** [comment]
On the target xrefs - what we have looks good, but there is an additional section that we should add for cross references that are mapped to the target rather than the target component e.g., links to the EBI Complex Portal (see CHEMBL2094108). This info is included in the target web service, but not sure if already in elastic? If there are any target xrefs, then I would add an extra section (probably at the start) called 'Target Cross References' with these. Leave all the other xref sections as they are.

**Comment by agaulton** [comment]
Target xrefs looks good. Can close that
